sgt-back-end/index.js

```diff
@@ -0,0 +1,128 @@
+const express = require('express');
+const pg = require('pg');
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+const app = express();
+
+app.use(express.json());
+
+app.get('/api/grades', (req, res) => {
+  const sql = `
+    select *
+    from grades;
+  `;
+
+  db.query(sql)
+    .then(result => {
+      const grades = result.rows;
+      res.status(200).json(grades);
+    })
+    // eslint-disable-next-line node/handle-callback-err
+    .catch(error => {
+      res.status(500).json({ error: 'An unexpected error occurred' });
+    });
+
+});
+
+app.post('/api/grades', (req, res) => {
+  const { name, course, score } = req.body;
+
+  const sql = `
+    insert into "grades" ("nme","course","score")
+    values ('${name}','${course}','${score}')
+    returning *;
+  `;
+  if (!name || !course || !score || score < 0 || score > 100) {
+    res.status(400).json({ error: 'Name, Course, and Score are required' });
+  }
+
+  db.query(sql).then(result => {
+    res.status(200).json({ name, course, score });
```

Comment:
Here you want to send back the data returned from the database, so you would want to send back `result.rows[0]`. You want to include the `gradeId` that the database generated in the data sent back to the client.